pytket/extensions/qujax/qujax_convert.py

```diff
@@ -223,20 +155,13 @@ def print_circuit_symbolic(
     sep_length: int = 1,
 ) -> List[str]:
     """
-    Returns and prints basic string representation of circuit
-    with symbolic parameters.
-
-    Note that the behaviour of print_circuit(circuit)
-    is different to print_circuit_symbolic(circuit),
-    print_circuit_symbolic will look for parameters in circuit.free_symbols()
-    and if there are none it will assume that none of the gates require parameters.
-    On the other hand, print_circuit will work out which gates are parameterised
-    based on e.g. circuit.get_commands()[0].op.params
+    Returns and prints basic string representation of circuit.
 
     :param circuit: Circuit to be converted.
     :type circuit: pytket.Circuit
-    :param symbol_map: dict that maps elements of circuit.free_symbols() (sympy)
-        to parameter indices.
+    :param symbol_map:
```

Comment:
You could refer here to the more information containing docs of the other two functions about this parameter?

Reply:
Good idea, amended now